specs/commandline/plugin.md

```diff
@@ -80,7 +82,7 @@ Aliases:
 
 ### Install a plugin from file system
 
-Install a Notation plugin from file system. Plugin file supports `.zip` and `.tar.gz` format. The checksum validation is optional for this case. 
+Install a Notation plugin from file system. Plugin file supports `.zip`, `.tar.gz`, and single plugin executable file format. The checksum validation is optional for this case. 
```

Comment:
```suggestion
Install a Notation plugin from the file system. When installing a plugin from a remote location, both `.zip` and `.tar.gz` file formats are supported. Additionally, installing a plugin from an executable file on the host system is also supported. In this scenario, checksum validation is optional.
```

Reply:
@priteshbandi This paragraph is for `installing from file system` only. Updated both `install from file system` and `install from URL` based on your comments.